lib/dockerregistry/inventory_test/TestParseManifestsFromDir/basic/a/manifest.yaml

```diff
@@ -0,0 +1,19 @@
+registries:
+- name: gcr.io/foo-staging
+  service-account: sa@robot.com
+  src: true
+- name: us.gcr.io/some-prod
+  service-account: sa@robot.com
+- name: eu.gcr.io/some-prod
+  service-account: sa@robot.com
+- name: asia.gcr.io/some-prod
+  service-account: sa@robot.com
+images:
+- name: foo-controller
+  dmap:
+    "sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa": ["1.0"]
+renames:
```

Comment:
Renames feels like it isn't worth the complexity?  Does anyone want to use it for anything more than a prefix?

Reply:
This will probably be simplified with #71 which addresses the common use case around renames.

I agree that renames add a lot of complexity, but I think with the new logic around PromotionEdges the renames are simpler than before.